src/renderer/views/Channel/Channel.js

```diff
@@ -2018,6 +2036,35 @@ export default defineComponent({
       })
     },
 
+    async saveStateInRouter(query) {
+      this.skipRouteChangeWatcherOnce = true
+      if (query === '') {
+        await this.$router.replace({ path: `/channel/${this.id}` }).catch(failure => {
+          if (isNavigationFailure(failure, NavigationFailureType.duplicated)) {
+            return
+          }
+
+          throw failure
+        })
+        return
+      }
+
+      await this.$router.replace({
+        path: `/channel/${this.id}`,
+        query: {
+          currentTab: 'search',
```

Comment:
Erm why is this setting the current tab in the query parameters, when it's a path parameter?